diff --git a/config.mk b/config.mk
index 0ec9e47..feaac15 100644
--- a/config.mk
+++ b/config.mk
@@ -12,7 +12,7 @@ X11LIB = /usr/X11R6/lib
 
 # includes and libs
 INCS = -I. -I/usr/include -I${X11INC}
-LIBS = -L/usr/lib -lc -L${X11LIB} -lX11
+LIBS = -L/usr/lib -lc -L${X11LIB} -lX11 -lasound
 
 # flags
 CPPFLAGS = -DVERSION=\"${VERSION}\" -D_DEFAULT_SOURCE
diff --git a/dwmstatus.c b/dwmstatus.c
index d2a4b03..6ac227b 100644
--- a/dwmstatus.c
+++ b/dwmstatus.c
@@ -16,10 +16,9 @@
 #include <sys/wait.h>
 
 #include <X11/Xlib.h>
+#include "./dwmstatus-volume.c"
 
 char *tzargentina = "America/Buenos_Aires";
-char *tzutc = "UTC";
-char *tzberlin = "Europe/Berlin";
 
 static Display *dpy;
 
@@ -80,7 +79,6 @@ setstatus(char *str)
 	XStoreName(dpy, DefaultRootWindow(dpy), str);
 	XSync(dpy, False);
 }
-
 char *
 loadavg(void)
 {
@@ -154,8 +152,6 @@ getbattery(char *base)
 		status = '-';
 	} else if(!strncmp(co, "Charging", 8)) {
 		status = '+';
-	} else {
-		status = '?';
 	}
 
 	if (remcap < 0 || descap < 0)
@@ -183,40 +179,34 @@ main(void)
 	char *bat;
 	char *bat1;
 	char *tmar;
-	char *tmutc;
-	char *tmbln;
-	char *t0, *t1, *t2;
+    int volume;
 
 	if (!(dpy = XOpenDisplay(NULL))) {
 		fprintf(stderr, "dwmstatus: cannot open display.\n");
 		return 1;
 	}
 
-	for (;;sleep(60)) {
+	for (;;sleep(1)) {
 		avgs = loadavg();
 		bat = getbattery("/sys/class/power_supply/BAT0");
-		bat1 = getbattery("/sys/class/power_supply/BAT1");
-		tmar = mktimes("%H:%M", tzargentina);
-		tmutc = mktimes("%H:%M", tzutc);
-		tmbln = mktimes("KW %W %a %d %b %H:%M %Z %Y", tzberlin);
-		t0 = gettemperature("/sys/devices/virtual/hwmon/hwmon0", "temp1_input");
-		t1 = gettemperature("/sys/devices/virtual/hwmon/hwmon2", "temp1_input");
-		t2 = gettemperature("/sys/devices/virtual/hwmon/hwmon4", "temp1_input");
-
-		status = smprintf("T:%s|%s|%s L:%s B:%s|%s A:%s U:%s %s",
-				t0, t1, t2, avgs, bat, bat1, tmar, tmutc,
-				tmbln);
+        bat1 = getbattery("/sys/class/power_supply/BAT1");
+		tmar = mktimes("%H:%M:%S %d %b", tzargentina);
+		volume = get_vol();
+
+		status = smprintf(
+			"V: %d% | L: %s | B: %s - %s | %s ",
+			volume,
+			avgs,
+			bat,
+            bat1,
+			tmar
+		);
 		setstatus(status);
 
-		free(t0);
-		free(t1);
-		free(t2);
 		free(avgs);
 		free(bat);
 		free(bat1);
 		free(tmar);
-		free(tmutc);
-		free(tmbln);
 		free(status);
 	}
 
