diff --git a/config.def.h b/config.def.h
index a2ac963..ccf71a5 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,21 +5,44 @@ static const unsigned int borderpx  = 1;        /* border pixel of windows */
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int showbar            = 1;        /* 0 means no bar */
 static const int topbar             = 1;        /* 0 means bottom bar */
-static const char *fonts[]          = { "monospace:size=10" };
-static const char dmenufont[]       = "monospace:size=10";
-static const char col_gray1[]       = "#222222";
-static const char col_gray2[]       = "#444444";
-static const char col_gray3[]       = "#bbbbbb";
-static const char col_gray4[]       = "#eeeeee";
-static const char col_cyan[]        = "#005577";
+static const char *fonts[]          = { "monospace:size=18" };
+static const char dmenufont[]       = "monospace:size=18";
+/* yin colors from yin-yang colorscheme */
+// static const char col_blackest[]    = "#080808"; [> blackest - 232 <]
+static const char col_black[]       = "#1c1c1c"; /* black - 234 */
+static const char col_gray01[]      = "#262626"; /* gray01 - 235 */
+// static const char col_gray02[]      = "#444444"; [> gray02 - 238 <]
+static const char col_gray03[]      = "#4e4e4e"; /* gray03 -239 */
+// static const char col_gray04[]      = "#585858"; [> gray04 -240 <]
+// static const char col_gray05[]      = "#666666"; [> gray05 - 242 <]
+// static const char col_gray06[]      = "#767676"; [> gray06 - 243 <]
+static const char col_gray07[]      = "#808080"; /* gray07 - 244 */
+// static const char col_gray08[]      = "#8a8a8a"; [> gray08 - 245 <]
+// static const char col_gray09[]      = "#949494"; [> gray09 - 246 <]
+// static const char col_gray10[]      = "#9e9e9e"; [> gray10 - 247 <]
+// static const char col_gray11[]      = "#a8a8a8"; [> gray11 - 248 <]
+static const char col_gray12[]      = "#b2b2b2"; /* gray12 - 249 */
+// static const char col_gray13[]      = "#bcbcbc"; [> gray13 - 250 <]
+// static const char col_gray14[]      = "#c6c6c6"; [> gray14 - 251 <]
+// static const char col_gray15[]      = "#e4e4e4"; [> gray15 - 254 <]
+// static const char col_white[]       = "#eeeeee"; [> white - 255 <]
+
+// static const char col_purple[]      = "#5f5fd7"; [> purple - 62 <]
+// static const char col_brown[]       = "#875f00"; [> brown - 94 <]
+static const char col_blue[]        = "#005f87"; /* blue - 24 */
+// static const char col_light[]       = "#00afff"; [> light blue - 39 <]
+// static const char col_green[]       = "#00875f"; [> green - 29 <]
+// static const char col_red[]         = "#870000"; [> red - 88 <]
+// static const char col_magenta[]     = "#87005f"; [> magenta - 89 <]
+
 static const char *colors[][3]      = {
 	/*               fg         bg         border   */
-	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
-	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
+	[SchemeNorm] = { col_gray07, col_black, col_black },
+	[SchemeSel]  = { col_gray12, col_black, col_gray01 },
 };
 
 /* tagging */
-static const char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
+static const char *tags[] = { "1", "2", "3", "4", "5" };
 
 static const Rule rules[] = {
 	/* xprop(1):
@@ -28,7 +51,7 @@ static const Rule rules[] = {
 	 */
 	/* class      instance    title       tags mask     isfloating   monitor */
 	{ "Gimp",     NULL,       NULL,       0,            1,           -1 },
-	{ "Firefox",  NULL,       NULL,       1 << 8,       0,           -1 },
+	// { "Firefox",  NULL,       NULL,       1 << 8,       0,           -1 },
 };
 
 /* layout(s) */
@@ -45,7 +68,8 @@ static const Layout layouts[] = {
 };
 
 /* key definitions */
-#define MODKEY Mod1Mask
+// Mod4Mask is the Super key
+#define MODKEY Mod4Mask
 #define TAGKEYS(KEY,TAG) \
 	{ MODKEY,                       KEY,      view,           {.ui = 1 << TAG} }, \
 	{ MODKEY|ControlMask,           KEY,      toggleview,     {.ui = 1 << TAG} }, \
@@ -57,13 +81,29 @@ static const Layout layouts[] = {
 
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
+static const char *dmenucmd[] = {
+    "dmenu_run",
+    "-m", dmenumon,
+    "-fn", dmenufont,
+    "-nb", col_gray01,
+    "-nf", col_gray07,
+    "-sb", col_blue,
+    "-sf", col_gray12,
+    NULL
+};
 static const char *termcmd[]  = { "st", NULL };
+static const char *mutecmd[] = { "amixer", "set", "Master", "toggle", NULL };
+static const char *volumeup[] = { "amixer", "set", "Master", "5%+", NULL };
+static const char *volumedown[] = { "amixer", "set", "Master", "5%-", NULL };
+static const char *lockscreen[] = { "slock", NULL };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
-	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
+	{ MODKEY,                       XK_space,  spawn,          {.v = dmenucmd } },
 	{ MODKEY|ShiftMask,             XK_Return, spawn,          {.v = termcmd } },
+	{ MODKEY|ShiftMask,             XK_a,      spawn,          {.v = volumedown } },
+	{ MODKEY|ShiftMask,             XK_s,      spawn,          {.v = volumeup } },
+	{ MODKEY|ShiftMask,             XK_d,      spawn,          {.v = mutecmd } },
 	{ MODKEY,                       XK_b,      togglebar,      {0} },
 	{ MODKEY,                       XK_j,      focusstack,     {.i = +1 } },
 	{ MODKEY,                       XK_k,      focusstack,     {.i = -1 } },
@@ -71,20 +111,22 @@ static Key keys[] = {
 	{ MODKEY,                       XK_d,      incnmaster,     {.i = -1 } },
 	{ MODKEY,                       XK_h,      setmfact,       {.f = -0.05} },
 	{ MODKEY,                       XK_l,      setmfact,       {.f = +0.05} },
+	{ MODKEY|ShiftMask,             XK_l,      spawn,          {.v = lockscreen } },
 	{ MODKEY,                       XK_Return, zoom,           {0} },
 	{ MODKEY,                       XK_Tab,    view,           {0} },
 	{ MODKEY|ShiftMask,             XK_c,      killclient,     {0} },
 	{ MODKEY,                       XK_t,      setlayout,      {.v = &layouts[0]} },
 	{ MODKEY,                       XK_f,      setlayout,      {.v = &layouts[1]} },
 	{ MODKEY,                       XK_m,      setlayout,      {.v = &layouts[2]} },
-	{ MODKEY,                       XK_space,  setlayout,      {0} },
-	{ MODKEY|ShiftMask,             XK_space,  togglefloating, {0} },
+	{ MODKEY,                       XK_p,      setlayout,      {0} },
+	{ MODKEY|ShiftMask,             XK_p,      togglefloating, {0} },
 	{ MODKEY,                       XK_0,      view,           {.ui = ~0 } },
 	{ MODKEY|ShiftMask,             XK_0,      tag,            {.ui = ~0 } },
 	{ MODKEY,                       XK_comma,  focusmon,       {.i = -1 } },
 	{ MODKEY,                       XK_period, focusmon,       {.i = +1 } },
 	{ MODKEY|ShiftMask,             XK_comma,  tagmon,         {.i = -1 } },
 	{ MODKEY|ShiftMask,             XK_period, tagmon,         {.i = +1 } },
+    { ControlMask|ShiftMask,        XK_3,      spawn,          SHCMD("scrot")},
 	TAGKEYS(                        XK_1,                      0)
 	TAGKEYS(                        XK_2,                      1)
 	TAGKEYS(                        XK_3,                      2)
