diff --git a/config.def.h b/config.def.h
index 9855e21..038a77f 100644
--- a/config.def.h
+++ b/config.def.h
@@ -1,11 +1,11 @@
 /* user and group to drop privileges to */
-static const char *user  = "nobody";
-static const char *group = "nogroup";
+static const char *user  = "hehe";
+static const char *group = "hehe";
 
 static const char *colorname[NUMCOLS] = {
 	[INIT] =   "black",     /* after initialization */
-	[INPUT] =  "#005577",   /* during input */
-	[FAILED] = "#CC3333",   /* wrong password */
+	[INPUT] =  "#5f5fd7",   /* during input */
+	[FAILED] = "#870000",   /* wrong password */
 };
 
 /* treat a cleared input like a wrong password (color) */
