diff --git a/config.def.h b/config.def.h
index 93a875a..fc3a3a4 100644
--- a/config.def.h
+++ b/config.def.h
@@ -65,5 +65,19 @@ static const char unknown_str[] = "n/a";
  */
 static const struct arg args[] = {
 	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+    /* 800MHz 1% m1% */
+    { cpu_freq,             "%sHz ",    NULL },
+    { cpu_perc,             "%s%% ",    NULL },
+    { ram_perc,             "m %s%% ",  NULL },
+    /* 🔊 <on/off> 75% */
+    { run_command,          "🔊 %s ",   "amixer get Master | grep 'Front Left: Playback' | awk '{ print $6 $5 }' | sed 's/\\[//g' | sed 's/\\]/ /' | sed 's/\\]//g'" },
+    /* 📶 95% */
+    { wifi_perc,            "📶 %s%% ", "wlp58s0" },
+    /* 🔋 97%- 100%? */
+    { battery_perc,         "🔋 %s%%",  "BAT0" },
+    { battery_state,        "%s ",      "BAT0" },
+    { battery_perc,         "%s%%",     "BAT1" },
+    { battery_state,        "%s ",      "BAT1" },
+    /* 12/02 16:20 */
+	{ datetime,             "%s ",      "%d/%m %H:%M" },
 };
