diff --git a/batt.c b/batt.c
index 3d72888..b88e067 100644
--- a/batt.c
+++ b/batt.c
@@ -13,21 +13,9 @@ void
 battprint(char *buf, size_t len, int acon, int life)
 {
 	char c;
-	static int frame = 0;
+	c = acon ? '+' : '-';
 
-	c = acon ? '>' : '<';
-	if (!acon && life <= 5)
-		snprintf(buf, len, "%s", crit[frame++ % LEN(crit)]);
-	else if (life >= 80)
-		snprintf(buf, len, "[////]=");
-	else if (life >= 60)
-		snprintf(buf, len, "[///%c]=", c);
-	else if (life >= 40)
-		snprintf(buf, len, "[//%c%c]=", c, c);
-	else if (life >= 20)
-		snprintf(buf, len, "[/%c%c%c]=", c, c, c);
-	else
-		snprintf(buf, len, "[%c%c%c%c]=", c, c, c, c);
+    snprintf(buf, len, "%d%c", life, c);
 }
 
 #ifdef __OpenBSD__
diff --git a/config.def.h b/config.def.h
index ba73f27..356d8b9 100644
--- a/config.def.h
+++ b/config.def.h
@@ -2,17 +2,65 @@
 int delay = 1;
 
 struct ent ents[] = {
-	/* reorder/edit/remove these as you see fit */
-	{ .fmt = "[%s] ",	.read = mpdread,	.arg = &(struct mpdarg){ .host = NULL, .port = 0 } },
-	{ .fmt = "[%s] ",	.read = countread,	.arg = "/home/USER/Maildir/INBOX/new" },
-	{ .fmt = "[%s] ",	.read = mixread,	.arg = NULL },
-	{ .fmt = "[%s] ",	.read = loadread,	.arg = NULL },
-	{ .fmt = "[%s] ",	.read = cpuread,	.arg = "/sys/devices/system/cpu/cpu0/cpufreq/scaling_cur_freq" },
-	{ .fmt = "[%s°] ",	.read = tempread,	.arg = "/sys/class/hwmon/hwmon1/temp1_input" },
-	{ .fmt = "%s ",		.read = battread,	.arg = &(struct battarg){ .cap = "/sys/class/power_supply/BAT0/capacity", .ac = "/sys/class/power_supply/AC/online" } },
-	{ .fmt = "%s ",		.read = wifiread,	.arg = NULL },
-	{ .fmt = "[%s] ",	.read = xkblayoutread,	.arg = NULL },
-	{ .fmt = "%s",		.read = keyread,	.arg = &(struct keyarg){ .sym = XK_Caps_Lock, .on = "[caps] ", .off = "" } },
-	{ .fmt = "%s ",		.read = fileread,	.arg = "/etc/myname" },
-	{ .fmt = "%s",		.read = dateread,	.arg = &(struct datearg){ .fmt = "%a %d %b %Y %H:%M %Z", .tz = NULL } },
+    /* reorder/edit/remove these as you see fit */
+    {
+        .fmt = "%s ",
+        .read = loadread,
+        .arg = NULL,
+    },
+    {
+        .fmt = "%s ",
+        .read = cpuread,
+        .arg = "/sys/devices/system/cpu/cpu0/cpufreq/scaling_cur_freq",
+    },
+    {
+        .fmt = "%s° ",
+        .read = tempread,
+        .arg = "/sys/class/hwmon/hwmon0/temp1_input",
+    },
+    {
+        .fmt = "🔊%s ",
+        .read = mixread,
+        .arg = NULL,
+    },
+    {
+        .fmt = "🔋%s ",
+        .read = battread,
+        .arg = &(struct battarg) {
+            .cap = "/sys/class/power_supply/BAT0/capacity",
+            .ac = "/sys/class/power_supply/AC/online",
+        },
+    },
+    {
+        .fmt = "🔋%s ",
+        .read = battread,
+        .arg = &(struct battarg) {
+            .cap = "/sys/class/power_supply/BAT1/capacity",
+            .ac = "/sys/class/power_supply/AC/online",
+        },
+    },
+    {
+        .fmt = "📶%s ",
+        .read = wifiread,
+        .arg = NULL,
+    },
+    {
+        .fmt = "%s",
+        .read = keyread,
+        .arg = &(struct keyarg) {
+            .sym = XK_Caps_Lock,
+            .on = "[caps] ",
+            .off = "",
+        },
+    },
+    {
+        .fmt = "%s",
+        .read = dateread,
+        .arg = &(struct datearg) {
+            .fmt = "%d/%m %H:%M",
+            .tz = NULL,
+        },
+    },
+    // { .fmt = "[%s] ", .read = xkblayoutread, .arg = NULL, },
+    // { .fmt = "[%s] ", .read = mpdread, .arg = &(struct mpdarg) { .host = NULL, .port = 0, }, },
 };
diff --git a/mix.c b/mix.c
index e765734..53ff4c8 100644
--- a/mix.c
+++ b/mix.c
@@ -216,7 +216,8 @@ mixread(void *arg, char *buf, size_t len)
 		warnx("snd_mixer_open: %s", snd_strerror(r));
 		return -1;
 	}
-	r = snd_mixer_attach(mixerp, "default");
+    // 'default' was not working here
+	r = snd_mixer_attach(mixerp, "hw:0");
 	if (r < 0) {
 		warnx("snd_mixer_attach: %s", snd_strerror(r));
 		goto out;
diff --git a/wifi.c b/wifi.c
index 4f2d2e7..01730db 100644
--- a/wifi.c
+++ b/wifi.c
@@ -11,19 +11,7 @@
 void
 wifiprint(char *buf, size_t len, int quality)
 {
-	char *icon;
-
-	if (quality == 100)
-		icon = "::";
-	else if (quality >= 75)
-		icon = ":.";
-	else if (quality >= 50)
-		icon = "..";
-	else if (quality >= 25)
-		icon = ". ";
-	else
-		icon = "  ";
-	snprintf(buf, len, "%s", icon);
+	snprintf(buf, len, "%d", quality);
 }
 
 #ifdef __OpenBSD__
